### PR TITLE
added space between home-left and home-right for better user visibility

### PR DESCRIPTION
**Pull request by Ankit895**
From 
Screenshot from 2020-03-31 17-41-51

to 
Screenshot from 2020-03-31 17-41-45

for better user visibility so that user can easily read.

### CODE REVIEW

**Comment by vramana** [comment]
@Ankit895 There are a lot of formatting changes can you undo them, and put them in a different commit or PR. @jeremyphilemon which is better?

**Comment by Ankit895** [comment]
I will update the pr

**Comment by Ankit895** [comment]
@vramana I have update the code, you can review and merge it

**Comment by vramana** [comment]
@Ankit895 It seems it just fixes the problem for your desktop screen. If you shrink the browser window a little more the problem will appear again. I don't think your fix is the correct one. What do you think?

1230 px viewport width

image

**Comment by Ankit895** [comment]
@vramana Thanks bro for bringing this to notice, updated the PR. Please let me know if you find any other issue

File: src/App.scss [hunk 1]
```diff
@@ -223,14 +223,18 @@ h6 {
   flex-direction: row;
   justify-content: space-around;
   flex-wrap: wrap;
-  margin-left: 10rem;
-  margin-right: 10rem;
+  margin-left: 8rem;
+  margin-right: 8rem;
```
Comment (vramana) [inline src/App.scss:227]: Is this change even necessary with the margin-left ?

Comment (Ankit895) [reply src/App.scss:227]: Yes, we need to decrease the margin from home else how would we give margin-left as there is no space to accommodate margin-left

Comment (vramana) [reply src/App.scss:227]: Did you check it? From what I see it's not necessary.

Comment (Ankit895) [reply src/App.scss:227]: Yes, I have checked it

**Comment by Ankit895** [comment]
@jeremyphilemon can you please review this?

**Comment by jeremyphilemon** [comment]
Thanks for the PR! I went ahead and patched it using a different strategy since settings margins wouldn't be of much help for devices having larger resolutions anyway.

**Pull request closed without merging**